compiler/dyno/include/chpl/uast/Formal.h

```diff
@@ -98,6 +98,7 @@ class Formal final : public VarLikeDecl {
    */
   Intent intent() const { return (Intent)((int)storageKind()); }
 
+  static const char* intentToString(Intent intent);
```

Comment:
Where is this implemented? I don't see it anywhere, but I do see a call to it in `AnonFormal` above.

Reply:
Wow, weird...why don't I get a linker error?